cedar-policy/src/frontend/is_authorized.rs

```diff
@@ -305,27 +303,35 @@ impl TryFrom<PartialResponse> for InterfaceResidualResponse {
 #[cfg_attr(feature = "wasm", derive(tsify::Tsify))]
 #[cfg_attr(feature = "wasm", tsify(into_wasm_abi, from_wasm_abi))]
 pub enum AuthorizationAnswer {
-    /// Represents a failure to parse and call the authorizer
-    Failure {
-        /// Parsing errors
+    /// Represents a failure to parse or call the authorizer entirely
+    ParseFailed {
```

Comment:
I prefer the more general name `Failure`. I'm imagining that we'll want to use these types for a general DRT interface, and other implementations of Cedar (e.g., the Lean) may return errors for more than just JSON parsing issues.

Reply:
That's fair.  I changed the name to `ParseFailed` as part of the merge from `main` (a recent commit to `main` changed the name to `ParseFailed` -- see https://github.com/cedar-policy/cedar/blob/38754319c7cd3b50083f44c4098cfb10f27d540c/cedar-policy/src/frontend/is_authorized.rs#L324), but I am happy to change back to `Failure` for the reasons you give

Reply:
Changed